docs/tools/offline/cli.md

```diff
@@ -74,51 +74,61 @@ Mbed CLI requires adding the following to the system `PATH`:
 * The Mbed CLI configuration command.
-* Setting an environment variable.
 * Adding the compiler's directory to your PATH.
-
-Methods for configuring toolchains that appear earlier in the above list override methods that appear later.
-
-##### Through `mbed_settings.py`
-
-Edit `mbed_settings.py` to set your toolchain:
-
-* To use the [Arm Compiler toolchain](https://developer.arm.com/products/software-development-tools/compilers/arm-compiler-5/downloads), set `ARM_PATH` to the *base* directory of your Arm Compiler installation (example: C:\Program Files\ARM\armcc5.06). The recommended version of the Arm Compiler toolchain is 5.06.
-* To use the [GNU Arm Embedded toolchain (GCC) version 6](https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads), set `GCC_ARM_PATH` to the *binary* directory of your GCC Arm installation (example: C:\Program Files\GNU Tools ARM Embedded\6 2017q2\bin). Use version 6 of GCC Arm Embedded; version 5.0 or any older version might be incompatible with the tools.
-* To use the [IAR EWARM toolchain](https://www.iar.com/iar-embedded-workbench/#!?architecture=ARM), set `IAR_PATH` to the *base* directory of your IAR installation (example: C:\Program Files (x86)\IAR Systems\Embedded Workbench 7.5\arm). Use versions 7.70 to 7.80.x of the IAR EWARM; newer (or older) versions might be incompatible with the tools.
-
-<span class="notes">**Note:** Because `mbed_settings.py` contains local settings (possibly relevant only to a single OS on a single machine), you should not check it into version control.</span>
+* Setting an environment variable.
+* The `mbed_settings.py` file in the root of your program. The tools will automatically create this file if it doesn't already exist.
 
 ##### Through Mbed CLI configuration
 
-You can set the Arm Compiler 5 location via the command:
+Mbed CLI stores it's own configuration about compiler locations both in project local settings, and user wide "global" settings. These settings may be set and viewed with the `mbed config` command. For example, you set the Arm Compiler 5 location for your user via the command:
 
 ```
 $ mbed config -G ARM_PATH "C:\Program Files\ARM"
 [mbed] C:\Program Files\ARM now set as global ARM_PATH
 ```
+The `-G` switch tells Mbed CLI to set this as a global setting, allowing all projects owned by this user to compile with Arm Compiler 5. You may instead set the `ARM_PATH` using a project local setting by omitting the `-G` command line switch.
+
+Mbed CLI supports a setting for each toolchain path. These setting are listed below, along with a description of what path is expected in each setting.
 
-The `-G` switch tells Mbed CLI to set this as a global setting, rather than local for the current program.
+* `ARM_PATH`: The path to the *base* directory of your Arm Compiler installation. This should be the directory containing the directory containing the binaries for `armcc` and friends.
```

Comment:
Query: The directory containing the binaries? Or the directory containing the directory containing the binaries?

Reply:
For this one, if the binary is `<some-path>/arm/bin/armcc` then the path should be `<some-path>/arm`. Do you think an example would help here?

Reply:
Yes